src/Accounts/Authentication/Config/ConfigInitializer.cs

```diff
@@ -190,6 +190,12 @@ private void RegisterConfigs(IConfigManager configManager)
                 true,
                 AzurePSDataCollectionProfile.EnvironmentVariableName,
                 new[] { AppliesTo.Az }));
+            configManager.RegisterConfig(new SimpleTypedConfig<bool>(
```

Comment:
We should also wrap this `RegisterConfig()` call with `#if`, because we don't want it exposed to customers.